requirements.txt

```diff
@@ -3,3 +3,6 @@ biopython
 openbabel
 # For some reason PyQuante can't be installed from PyPI.
 # PyQuante
+
+# for unit tests
+mock==2.0.0 ; python_version < '3.3'
```

Comment:
I don't understand the syntax. Does this mean only install mock for Python versions < 3.3, or something else?

Reply:
[Yes, apparently it's called an "environment marker"](https://www.python.org/dev/peps/pep-0496/)

In python 3.4 and later `mock` is in the standard library as `unittest.mock`

Reply:
I just looked at travis logs, and it never does `pip install -r requirements.txt` and all dependencies are listed individually in `.travis.yml`. Is this because of the problems with pyquante?

Reply:
I was probably using it for a while, discovered the versioning issue, and just left the file behind.

I may also have tried `python_version` in the past and I just didn't remember, and older versions of pip had trouble with it?

One comment is that PyQuante requires 2.7.

If you can get this to work on all our testing versions, that would simplify Travis immensely.

Reply:
PR #467